### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by skullydazed** [mqtt-smarthome/mqtt-smarthome#39]
Last update seems to be 3 years ago, there's no mention of things like esphome or tasmota, and the slack link is invalid. I really like this concept but it's hard to find some of the central pieces I need (like a centralized state service) so I'm hoping people are still working on it.

**Comment by xAPPO** [comment]
Seems to be dead

**Comment by EternityForest** [comment]
I really like the concept but it doesn't seem like there is any actual software available for it.   If anyone is still interested, I'd be up for working on it.

I'm developing my own web based HA here that relies on MQTT for a lot of stuff, that could serve as the central state with modifications. https://github.com/EternityForest/KaithemAutomation

I'm using it in production, but it's gotten to be a bit more than I would really like to manage on my own, and I'd like to make it more modular.


I do worry that this is *too* modular.    I would not want to actually use a system with bazillions of different pieces to set up.  I also would hope to see something beyond *just* a bare minimum lowest common denominator kind of thing.

You might run into issues if you didn't want to send secrets via the bus, but what if we extended the protocol a bit to cover saving and managing the whole configuration?

That way things would not need anything beyond just the MQTT connection parameters or even a writable disk for anything but caching.


It would also mean you could write a completely standalone but fully integrated system, with its own web UI and everything, but still be able to manage all your state in one place.   Things like ESP32 nodes could persist state independently while being centrally managed.

**Comment by hobbyquaker-agent** [comment]
Yes — the convention has been picked up again. In September 2026 it got its 2.0 revision ([SPEC.md](https://github.com/mqtt-smarthome/mqtt-smarthome/blob/master/SPEC.md)) and a new [README](https://github.com/mqtt-smarthome/mqtt-smarthome#readme) with a list of software that is maintained today, including a family of interfaces built on a shared Node.js library ([mqtt-interfaces-core](https://github.com/hobbyquaker/mqtt-interfaces-core)). The dead Slack invite is gone; issues and pull requests here are the way to reach us. ESPHome, zigbee2mqtt, Node-RED and Home Assistant are named in the README as projects that work well on the same broker.

About a centralized state service: in mqtt-smarthome the broker is that service — every interface publishes its state retained, so any client that connects gets the current state of everything, and `+/info` lists the running interfaces. @EternityForest's configuration idea is discussed in #40.

Closing this as answered.

---
*Posted by Claude (Anthropic) on behalf of @hobbyquaker.*

**Comment by skullydazed** [comment]
@hobbyquaker Thanks for the update, and it's nice to see some progress here. I don't think this will be relevant to me moving forward. Not because I've stopped working on MQTT based smarthome services, but because I've long since built a bunch of software around conventions I decided. I would love to help with a community developed standard, but it seems you aren't interested in collaborating with others on it. Best of luck with your project, I hope you find success with it.
